iocore/net/P_UnixNet.h

```diff
@@ -222,33 +224,53 @@ struct PollCont : public Continuation {
 //
 class NetHandler : public Continuation, public EThread::LoopTailHandler
 {
+  using self_type = NetHandler; ///< Self reference type.
 public:
   // @a thread and @a trigger_event are redundant - you can get the former from the latter.
   // If we don't get rid of @a trigger_event we should remove @a thread.
-  EThread *thread = nullptr;
-  Event *trigger_event;
+  EThread *thread      = nullptr;
+  Event *trigger_event = nullptr;
   QueM(UnixNetVConnection, NetState, read, ready_link) read_ready_list;
   QueM(UnixNetVConnection, NetState, write, ready_link) write_ready_list;
   Que(UnixNetVConnection, link) open_list;
   DList(UnixNetVConnection, cop_link) cop_list;
   ASLLM(UnixNetVConnection, NetState, read, enable_link) read_enable_list;
   ASLLM(UnixNetVConnection, NetState, write, enable_link) write_enable_list;
   Que(UnixNetVConnection, keep_alive_queue_link) keep_alive_queue;
-  uint32_t keep_alive_queue_size;
+  uint32_t keep_alive_queue_size = 0;
   Que(UnixNetVConnection, active_queue_link) active_queue;
-  uint32_t active_queue_size;
-  uint32_t max_connections_per_thread_in;
-  uint32_t max_connections_active_per_thread_in;
-
-  // configuration settings for managing the active and keep-alive queues
-  uint32_t max_connections_in;
-  uint32_t max_connections_active_in;
-  uint32_t inactive_threashold_in;
-  uint32_t transaction_no_activity_timeout_in;
-  uint32_t keep_alive_no_activity_timeout_in;
-  uint32_t default_inactivity_timeout;
-
-  int startNetEvent(int event, Event *data);
+  uint32_t active_queue_size = 0;
+
+  /// configuration settings for managing the active and keep-alive queues
+  struct Config {
+    uint32_t max_connections_in                 = 0;
+    uint32_t max_connections_active_in          = 0;
+    uint32_t inactive_threshold_in              = 0;
+    uint32_t transaction_no_activity_timeout_in = 0;
+    uint32_t keep_alive_no_activity_timeout_in  = 0;
+    uint32_t default_inactivity_timeout         = 0;
+  };
```

Comment:
Why not:
enun ConfigSettings { ... , NUM_CONFIG_SETTINGS };
uint32_t config[NUM_CONFIG_SETTINGS];

Reply:
I considered that, but decided the changes were already large enough to be a concern. I'm not sure an enum would add much value here, since they the structures are by design identical.

Reply:
Given your casting below.. I agree with @ywkaras